pkg/queue/sqs.go

```diff
@@ -276,7 +295,7 @@ func (s *SQS) poll(key string, queueSpec QueueSpec) {
 					queueSpec.name, err)
 				return
 			} else {
-				klog.Fatalf("Unable to receive message from queue %q, %v.",
+				klog.Errorf("Unable to receive message from queue %q, %v.",
```

Comment:
I prefer it crash in these scenarios so that we get KubePodCrashLoop alert. These errors when fails is easier to fix.